### ISSUE TITLE
not authorized

### ISSUE THREAD

**Issue by rodrigoorrego** [tgalal/yowsup#2074]
yesterday I had it running... sent some test messages using the demos and yowsup-cli

but today I was going to use it to actually make a application... but now I'm having "not authorized" error...

if someone already gone through this or has some ideas...

what might it be? thanks in advance :)

--

yowsup-cli  v2.0.15
yowsup      v2.5.2

Copyright (c) 2012-2016 Tarek Galal
http://www.openwhatsapp.org

This software is provided free of charge. Copying and redistribution is
encouraged.

If you appreciate this software and you would like to support future
development please consider donating:
http://openwhatsapp.org/yowsup/donate

Authentication Error: not-authorized

**Comment by rodrigoorrego** [comment]
obs: just sent like... 6 messages yesterdar... to myself... and for testing purposes :(

**Comment by Simonwcm** [comment]
i also can't send message now.

**Comment by isaiasneto** [comment]
That happen to me too... Was working fine, after some days without using, wen I tried to login, got blocked!

**Comment by ceciliacsilva** [comment]
Hello,

I was using the yowsup client also this week and worked. Then I started to adapt the layer.py and run.py to recive and send messages and was working untill this morning. Since then I 'm reciving "status: fail" and "reason: blocked" when I try to register. And I think, because I'm no registered when I try to send a message I get an "Authentication error: not-authorized".

I also tried to register my number on a real phone and whatsapp has blocked me. Does anyone know how to generate a new md5_classes key?
